helm/hpcc/templates/roxie.yaml

```diff
@@ -242,6 +242,7 @@ spec:
       serviceAccountName: "hpcc-default"
       initContainers:
       {{- include "hpcc.changePlaneMountPerms" $commonCtx | indent 6 }}
+      terminationGracePeriodSeconds: 100        
```

Comment:
is 100 seconds always enough?
Should it be configurable and propagate through, so that "agentQueryReleaseDelaySeconds" is based on it?